src/Plivo/Resources/Message/MessageInterface.php

```diff
@@ -144,66 +132,101 @@ protected function getList($optionalArgs = [])
      * @throws PlivoValidationException,PlivoResponseException
      */
 
-    public function create($src, array $dst, $text=null,
-                           array $optionalArgs = [], $powerpackUUID = null)
+    public function __call($name_of_method, $arguments)
     {
-        $mandatoryArgs = [
-            'dst' => implode('<', $dst),
-        ];
+        if ($name_of_method == 'create' and count($arguments) == 1)
+        {
+            $arguments = $arguments[0];
+            $dest = explode('<', $arguments['dst']);
+            if (!array_key_exists('src', $arguments) && !array_key_exists('powerpackUUID', $arguments))
+            {
+                throw new PlivoValidationException("Specify either powerpack_uuid or src in request params to send a message.");
+            }
 
-        if (ArrayOperations::checkNull($mandatoryArgs)) {
-            throw new PlivoValidationException(
-                "Mandatory parameters cannot be null");
-        }
+            if (!array_key_exists('src', $arguments) && array_key_exists('powerpackUUID', $arguments))
+            {
+                throw new PlivoValidationException("Both powerpack_uuid and src cannot be specified. Specify either powerpack_uuid or src in request params to send a message.");
+            }
 
-        if (is_null($src) &&  is_null($powerpackUUID)) {
-            throw new PlivoValidationException(
-                "Specify either powerpack_uuid or src in request params to send a message."
-            );
+            foreach ($dest as $dst)
```

Comment:
if the destination param is array, can we keep same across all the sdk?

Reply:
Fixed.